### PR TITLE
Start writeup for streaming setup

### PR DESCRIPTION
**Pull request by domgreen**
Have started there are a few places to add more links to @thoeni can you add mic stuff.

Happy to change layout etc. Thought it would be best to get something started that we can iterate on in the future.

### CODE REVIEW

File: runbooks/youtube/setup.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Live Streaming Setup
+
+## Hardware
+- Speaker Laptop
+- Streaming Laptop
+- Power Cables
+- Video Adaptors for Hooking Streaming Laptop to venue AV System
```
Comment (myitcv) [inline runbooks/youtube/setup.md]: I think we should provide links to examples of all this stuff

**Comment by domgreen** [comment]
@thoeni mind casting an eye over this?

File: runbooks/youtube/setup.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- USB/USBc Hub
+  - [Multi-Port](https://www.amazon.co.uk/Multi-port-Adapter-Delivery-MacBook-Spectre-Grey/dp/B07CPRWL41/ref=sr_1_3?keywords=usbc+hub&qid=1568149666&s=computers&sr=1-3) 
+- Camera for Recording Speaker
+  - Logitech HD Pro Webcam C920 [Amazon](https://www.amazon.co.uk/Logitech-Calling-Recording-Microphones-Adjustable/dp/B006A2Q81M/ref=sr_1_3?keywords=Logitech+HD+Pro+Webcam+C920&qid=1568149155&s=gateway&sr=8-3)
+  - DSLR would also work for higher quality
+- A video capture passthrough device (used to stream slides to streaming laptop)
+  - [Mirbox](https://www.amazon.co.uk/MiraBox-Capture-30FPS-1080P-Passthrough/dp/B07R5T29KN/ref=sr_1_1?keywords=Mirabox+USB-C+Capture+Card%2C+4K+30FPS%2CHD+1080P+60FPS%2CHDMI+Game+Video+Capture+Card+with+Mic+Input+and+HDMI+Passthrough+-+Compatible+with+Windows%2C+Linux%2C+macOS%2C+OBS+Streaming+for+PS3+PS4+Xbox+Wii+U&linkCode=g13&qid=1566382602&s=computers&sr=1-1)
+  - [Cam Link](https://www.amazon.co.uk/Elgato-Cam-Link-Broadcast-AmazonBasics/dp/B07V6LNJWQ/ref=sr_1_3?keywords=elgato+camlink+4k&qid=1568149490&s=computers&sr=1-3)
```
Comment (thoeni) [inline runbooks/youtube/setup.md]: I don't think Cam Link does what we want... we want a card with HDMI input AND output...

Comment (domgreen) [reply runbooks/youtube/setup.md]: It does, I've used it at the Prometheus Meetup :)

**Pull request merged**